aikau/src/main/resources/alfresco/core/topics.js

```diff
@@ -72,6 +72,30 @@ define([],function() {
       CLEAR_SELECTED_ITEMS: "ALF_CLEAR_SELECTED_ITEMS",
 
       /**
+       * This can be published to make a request to create a dialog. It is typically handled by the
+       * [DialogService]{@link module:alfresco/services/DialogService}.
+       *
+       * @instance
+       * @type {string}
```

Comment:
I think it'd make sense for the topics constant file to define the event, but am happy for this to be done under a separate ticket, as part of a wider sweep.